torch/_inductor/pattern_matcher.py

```diff
@@ -1046,6 +1046,7 @@ def run_node(self, old_node):
             new_node = super().run_node(old_node)
             if isinstance(new_node, torch.fx.Proxy):
                 new_node.node.meta.update(old_node.meta)
+                new_node.node.name = old_node.name
```

Comment:
I think you should use this API instead to safe  https://github.com/pytorch/pytorch/blob/main/torch/fx/graph.py#L840